CONTRIBUTING.md

```diff
@@ -22,3 +23,18 @@ If you added a contributor by mistake, you can remove them in a comment with:
 
 If you are making a pull request on behalf of someone else but you had no part in designing the 
 feature, you can remove yourself with the above syntax.
+
+## Work Mode
+We invite all to contribute to the development of this charter by raising issues
+and submitting PRs. All proposed changes to the document must be agreed to by
+the listed chairs before merging. Other approvals may be sought, but are not
+required, particularly while this document is in a draft state.
+
+PRs will be merged at the chairs' discretion, and may be merged quickly. Please
+raise an issue if changes have been made to the draft charter that you do not
+agree with. However, please keep in mind that while we will strive to address
+issues in a manner that is acceptable to those have raised them, we cannot
+guarantee that all issues will result in changes satisfactory to all parties,
+nor that all parties will have sufficient time to review changes before they are
+merged into the draft.
```

Comment:
```suggestion
PRs will be merged at the chairs' discretion, and may be merged quickly. Please
raise an issue if changes have been made to the draft charter that you do not
agree with. However, please keep in mind that while we will strive to address
issues in a manner that is acceptable to those have raised them, we cannot
guarantee that all issues will result in changes satisfactory to all parties,
nor that all parties will have sufficient time to review changes before they are
merged into the draft.
```

The VCWG Charter is a VCWG work item, isn't it? Why are we not allowing VCWG members enough time to review PRs?

This creates an atmosphere where things can be rushed through, which could then just inevitably lead to objections when the charter goes out to review.

My suggestion is that PRs must be given time on at least one call for discussion, and a minimum of 7 days from the day they were raised, to be merged by the Chairs.

Reply:
I'd like to see the charter draft ready for AC review within a couple of weeks, which imo doesn't allow for review periods, 
I have added a line which I hope allays concerns that the chairs will submit a charter which does not meet with the approval to the AC for review.